pkg/cmd/roachtest/tests/restore.go

```diff
@@ -466,6 +469,9 @@ type hardwareSpecs struct {
 	// volumeSize indicates the size of per node block storage (pd-ssd for gcs,
 	// ebs for aws). If zero, local ssd's are used.
 	volumeSize int
+	// throughput is the min provisioned throughput of the storage, in MB/s.
+	// TODO(pavelkalinnikov): currently works only for EBS.
+	throughput int
```

Comment:
nit: rename to AWSEBSThoughput

Reply:
Renamed to `AWSVolumeThroughput`, to make it similar to `VolumeSize`, and to avoid the `AWSEBS` abbreviation hell.